src/ui.py

```diff
@@ -180,12 +180,11 @@ def add_entry(io, app: App):
     )
 
     # Add the entry to the Bibliography
-    err = app.add_entry(entry)
-    if err is None:
+    try:
+        app.add_entry(entry)
         io.print("Entry successfully added to the database.")
-    else:
-        print(err)
-
+    except TitleAlreadyExists as e:
+        print(e)
```

Comment:
@sevonj shouldn't we `io.print(e)` ?

Reply:
![image](https://github.com/sevonj/miniprojekti/assets/49162364/79bcc962-136a-4db9-930a-2ab19d88e306)

I think that's the only place we invoke `print()` in our software apart from the `AppIO.print()` implementation, it might be a leftover from when we migrated to said class